astropy/coordinates/earth.py

```diff
@@ -646,12 +651,45 @@ def get_itrs(self, obstime=None):
 
         # do this here to prevent a series of complicated circular imports
         from .builtin_frames import ITRS
-        return ITRS(x=self.x, y=self.y, z=self.z, obstime=obstime)
+        itrs_coo = ITRS(x=self.x, y=self.y, z=self.z, obstime=obstime)
```

Comment:
I wondered about this too, though perhaps it ends up being better to have something like an `Observer` analogue of a `SkyCoord`. Though since this would change API, perhaps best discussed outside of the PR.